caffe2/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ set(Caffe2_GPU_SRCS)
 set(Caffe2_CPU_TEST_SRCS)
 set(Caffe2_GPU_TEST_SRCS)
 
+# Caffe2_CPU_OBSERVER_SRCS is the list of files used for benchmarking purpose
+set(Caffe2_CPU_OBSERVER_SRCS)
```

Comment:
Is this ever needed at global scope?  I think the variable can exist exclusively in the `share` directory or even the `share/contrib/observers` directory

Reply:
Can you suggest how? After I move it to the top of the share directory, the build fails because the files are not found.